content/tutorials/build-apps/build-a-desktop-wallet-in-js.md

```diff
@@ -0,0 +1,211 @@
+
+For this tutorial, we're using basic HTML and pre-loaded CSS files. If you'd like to change the aesthetic, you can replace it with any CSS framework of your choice!
+
+8. Copy the code from the [index.css](_code-samples/build-a-wallet/js/index.css) file to your project's index.css file to use the pre-made stylings.
+
+### Step 3: Creating the Send XRP Page
+
+Till now, we have created the home page that displays the account and ledger details. In this step, we will create the Send XRP page that will allow users to send XRP to another account.
+
+![Send XRP Page Screenshot](img/js-wallet-send-xrp.png)
+
+1. Create a folder named send-xrp in the src directory.
+
+2. Add two files named send-xrp.js and send-xrp.html. You can also create a separate css file depending on your choice. 
+
+3. Copy the contents of the [send-xrp.html](_code-samples/build-a-wallet/js/src/send-xrp/send-xrp.html) file to your send-xrp.html file. The provided HTML code includes three input fields for the destination address, amount, and destination tag, each with their corresponding labels.
+
+4. Now that we have the HTML code, let's add the JavaScript code. In helpers folder, add a new file named submit-transaction.js and copy the code written below to the file.
+
+{{ include_code("_code-samples/build-a-wallet/js/src/helpers/submit-transaction.js", language="js") }}
+
+In the code above, we are using the [submit](https://xrpl.org/submit.html) method to submit the signed transaction to the XRPL. You can read more about the submit method [here](https://xrpl.org/submit.html).
+
+5. Now back to the send-xrp.js file, copy the code written below to the file.
+
+{{ include_code("_code-samples/build-a-wallet/js/src/send-xrp/send-xrp.js", language="js") }}
+
+In the code above, we retrieve elements from DOM, add event listeners, validate inputs and after validation, we call the `submitTransaction` function from the submit-transaction.js file to submit the transaction to the XRPL.
+
+### Step 4: Creating the Transactions Page
+
+Now that we have created the home page and the send XRP page, let's create the transactions page that will display the transaction history of the account.
+
+  1. Account: The account that sent the transaction.
+  2. Destination: The account that received the transaction.
+  3. Amount: The amount of XRP sent in the transaction.
+  4. Transaction Type: The type of transaction.
+  5. Result: The result of the transaction.
+  6. Link: A link to the transaction on the XRP Ledger Explorer.
+
+![Transactions Page Screenshot](img/js-wallet-transaction.png)
+
+1. Create a folder named transaction-history in the src directory.
+
+2. Add two files named transaction-history.js and transaction-history.html.
+
+3. In the transaction-history.js file, copy the code written below.
+
+{{ include_code("_code-samples/build-a-wallet/js/src/transaction-history/transaction-history.js", language="js") }}
```

Comment:
Ditto on explaining the intent before copying the code.

Reply:
Added